dependencies/Dockerfile

```
@@ -13,29 +13,35 @@ WORKDIR /tmp/build-libraw
 COPY --chmod=0755 build_libraw.sh .
 RUN --mount=type=cache,target=${BUILD_CACHE_DIR},id=libraw-${TARGETARCH}-${LIBRAW_VERSION} \
     --mount=type=secret,id=github_token \
-    export GITHUB_TOKEN=$(cat /run/secrets/github_token) \
+    if [ -f /run/secrets/github_token ]; then \
```

`build_*.sh` doesn't use GITHUB_TOKEN now. Do you want to keep this for future use?

---

Actually, they use it (if available) for downloading tarballs from api.github.com to overcome the request rate limit:
https://github.com/photoview/photoview/blob/0192e4e64020f383d8a214707e1aee14e4a22bc7/dependencies/build_imagemagick.sh#L49
https://github.com/photoview/photoview/blob/0192e4e64020f383d8a214707e1aee14e4a22bc7/dependencies/build_libheif.sh#L41
https://github.com/photoview/photoview/blob/0192e4e64020f383d8a214707e1aee14e4a22bc7/dependencies/build_libraw.sh#L37
The download FFmpeg script doesn't use it, as it gets the file from the regular github.com, where there is no such limit.

I didn't change this code. I just made sure that it doesn't cause failures and silently skipped in the case of local runs, where the token is not provided.